spec/formatting.md

```diff
@@ -222,6 +222,74 @@ the following steps are taken:
 
    The form that resolved _operand_ and _option_ values take is implementation-defined.
 
+   A _local-declaration_ binds the resolved value of an _expression_
+   to a _variable_.
+   Thus, the output of one _function_ is potentially the _operand_
+   of another _function_,
+   or the value of one of the _options_ for another function.
+   For example, in
+   ```
+   .input {$n :number minIntegerDigits=3}
+   .local $n1 = {$n :number maxFractionDigits=3}
+   ```
+   the output of the first call to `:number`
+   is the input of the second call to `:number`.
```

Comment:
This is saying outputs and inputs, which, as I noted before, isn't correct.

Reply:
I tried to address this in [8a5f589](https://github.com/unicode-org/message-format-wg/pull/686/commits/8a5f5893a866a2bab49c0f64b92948c2ccab01f7)